items.json

```diff
@@ -62972,14 +62972,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_fullered_western_mace_v1_h0",
-    "baseId": "crpg_fullered_western_mace_v1",
+    "id": "crpg_fullered_western_mace_v2_h0",
```

Comment:
example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.